Misc/NEWS.d/next/Core_and_Builtins/2025-01-22-14-24-44.gh-issue-129149.wAYu43.rst

```diff
@@ -0,0 +1,2 @@
+Add missing fast path in :c:func:`PyLong_FromUnsignedLong`, :c:func:`PyLong_FromUnsignedLongLong` and :c:func:`PyLong_FromSize_t` functions
+for compact integers. And refactor :c:func:`PyLong_FromLong` and :c:func:`PyLong_FromLongLong` to use the macro ``PYLONG_FROM_SIGNED``.
```

Comment:
I'm not sure it's needed to know that the functions were refactored using a private macro. The code was only moved around right?

Reply:
I suspect that the change might affect performance.  That's something that should be documented.  We shouldn't mention the private macro.

Reply:
I'm not saying we should remove the mention to the missing fast path. I'm suggesting removing the refactoring information.

Reply:
And also revert the refactoring if it does not add new value to the current code. If the current code is already having a fast path, I would prefer doing it two different commits (easier to bisect in case of an issue)